terraform/aws/app-infrastructure/aws-prometheus-grafana/modules/grafana-dashboard/main.tf

```diff
@@ -49,6 +49,20 @@ resource "grafana_data_source" "prometheus" {
   })
 }
 
+##############################
```

Comment:
Please remove the added lines 52-65. The scope of this PR is to address the fluentbit parser